src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -6382,27 +6382,28 @@ bool Unit::HandleDummyAuraProc(Unit* victim, uint32 damage, AuraEffect* triggere
                 }
                 else
                 {    // Check Party/Raid Group
-                    if (Group* group = ToPlayer()->GetGroup())
```

Comment:
Actually, how can IsPet() return true here inside the else of https://github.com/Goatform/TrinityCore/blob/3.3.5/src/server/game/Entities/Unit/Unit.cpp#L6375 check ? Was this change tested ingame AND debugged with both cases IsPet() true/false ?